packages/gollm/tasks/interventions_from_dataset.py

```diff
@@ -0,0 +1,44 @@
+import json
+import sys
+
+from chains import interventions_from_dataset_chain
+from entities import InterventionsFromDataset
+from llms.openai.OpenAiTools import OpenAiTools
+from taskrunner import TaskRunnerInterface
+
+
+def cleanup():
+    pass
+
+
+def main():
+    exitCode = 0
+    try:
+        taskrunner = TaskRunnerInterface(description="Extract interventions from paper CLI")
+        taskrunner.on_cancellation(cleanup)
+
+        input_dict = taskrunner.read_input_dict_with_timeout()
+
+        taskrunner.log("Creating InterventionsFromDataset model from input")
+        input_model = InterventionsFromDataset(**input_dict)
+        amr = json.dumps(input_model.amr, separators=(",", ":"))
+
+        taskrunner.log("Sending request to OpenAI API")
+        llm = OpenAiTools()
+        response = interventions_from_dataset_chain(llm, dataset=input_model.dataset, amr=amr)
+        taskrunner.log("Received response from OpenAI API")
+
+        taskrunner.write_output_dict_with_timeout({"response": response})
+
+    except Exception as e:
+        sys.stderr.write(f"Error: {str(e)}\n")
```

Comment:
Per @mwdchang recommend adding the traceback for better debugging:
https://github.com/DARPA-ASKEM/terarium/pull/5768/commits/93e25bb0b50c27540fca89379917ca8da847a5bc

Reply:
`str(e)` is kinda just the string error message, traceback yields the callstack

Reply:
fixed, and then fixed all the other taskes